### PR TITLE
Add Noah to people page

### PR DESCRIPTION
**Pull request by noli3118**

### CODE REVIEW

**Review by joewie — CHANGES_REQUESTED** [review]
Image file seems to be empty.

**Comment by alecive** [comment]
@noli3118 as per instructions: the picture should be square! 

Also, please remove the black borders from it.

File: img/people/noah.jpg
Comment (alecive) [inline img/people/noah.jpg:1]: this file should be removed if you won't use it.

File: img/people/noah.png
Comment (alecive) [inline img/people/noah.png:1]: this file should be square , and you should remove the border.

File: _data/people.yml [hunk 1]
```diff
@@ -129,6 +129,12 @@
    email: frli5468@colorado.edu
    pic: freddy.png
 
+ - name: Noah Liska
+   role: undergrad
+   linkedin: noah-liska
+   email: noli3118@colorado.edu
+   pic: noah.jpg
```
Comment (alecive) [inline _data/people.yml:136]: if you are using the png, this should point to the png picture.

**Review by alecive — CHANGES_REQUESTED** [review]

**Comment by alecive** [comment]
Nice! Good job.

**Comment by alecive** [comment]
@noli3118 unfortunately, because this PR too too long, there are now conflicts in the `people.yml` file. Please synchronize with @calebescobedo and have him resolve them as these thigns are tricky.

File: _data/people.yml [hunk 2]
```diff
@@ -330,3 +339,4 @@
    role: alumni
    linkedin: william-wang-a445b4193
    pic: wwang.jpg
+>>>>>>> 2e45997d69a8389896705e633dc7f498d439376c
```
Comment (alecive) [inline _data/people.yml]: I see there are still conflicts. I recommend you close this PR, you `pull --rebase` the repo and open a new one. @calebescobedo

**Review by alecive — CHANGES_REQUESTED** [review]

**Pull request closed without merging**